index.js

```diff
@@ -52,6 +52,11 @@ GulpRunner.prototype.run = function(tasks, options, cb) {
       cb(null)
     }
   });
+
+  process.on('SIGINT', function() {
+    gulp.kill();
+    process.exit(1);
```

Comment:
I was going to add some tests, but I found it slightly hard to test; if that's something you'd like to see, I can take another stab!